atomic_reactor/core.py

```diff
@@ -38,8 +38,8 @@
         BUILD_JSON, DOCKER_SOCKET_PATH, DOCKER_MAX_RETRIES, DOCKER_BACKOFF_FACTOR,\
         DOCKER_CLIENT_STATUS_RETRY
 from atomic_reactor.source import get_source_instance_for
-from atomic_reactor.util import (
-    ImageName, clone_git_repo, figure_out_build_file, Dockercfg)
+from atomic_reactor.util import (ImageName, figure_out_build_file, Dockercfg)
```

Comment:
Now this is in a single line, we could remove the parenthesis

Reply:
Fixed.